prizetap/views.py

```diff
@@ -3,6 +3,7 @@
 from django.db.models import Case, F, When
 from django.shortcuts import get_object_or_404
 from django.utils import timezone
+from core.utils import RequestContextExtractor
```

Comment:
You are creating a duplicate instance by instantiating RequestContextExtractor within HasVerifiedCloudflareCaptcha

Reply:
this is not the case as im only instantiating `RequestContextExtractor` inside `views.py` in `tokenTap` and `prizetap`